src/workerd/api/crypto-impl-asymmetric.c++

```diff
@@ -730,6 +730,12 @@ kj::OneOf<jsg::Ref<CryptoKey>, CryptoKeyPair> CryptoKey::Impl::generateRsa(
       publicExponent.size(), nullptr), InternalDOMOperationError, "Error setting up RSA keygen.");
 
   auto rsaPrivateKey = OSSL_NEW(RSA);
+  // TODO: boringssl silently uses (modulusLength & ~127) for the key size, i.e. it rounds down to
+  // the closest multiple of 128 bits. This can easily cause confusion when non-standard key sizes
+  // are requested. Ideally we would throw an error when trying to create keys where the size would
+  // be rounded down, but this would likely break existing scripts.
```

Comment:
We could emit a log message to confirm whether we'd break anyone, like here: https://github.com/cloudflare/workerd/blob/0efe8aeee69d1a0ae3939fa0ccec15dd758ccd3a/src/workerd/io/worker.c%2B%2B#L377-L381

Reply:
We could also do the same for the other cases where you identify that we could theoretically be introducing a breaking change, e.g. when calling validateRsaParams() during RSA key import, and 62/63-byte RSASSA-PKCS1-v1_5 keys used with SHA-256.

Reply:
Sounds good, I'll try doing that. Can you explain why there's a lambda expression around this instance of KJ_LOG – is it related to how the warning is reported?

Reply:
Switched to log messages to detect potential breakage for key signing and validating imported keys. For the key size issue I'd expect that there likely are scripts using sizes other than multiples of 128, so I left the TODO message as-is for now.